internal/controller/utils/nim.go

```diff
@@ -273,7 +273,9 @@ func attemptToPullManifest(runtime NimRuntime, tokenResp *NimTokenResponse) erro
 	}
 
 	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", tokenResp.Token))
+	req.Header.Add("Accept", "application/vnd.oci.image.index.v1+json")
 
+	fmt.Println(req.Header.Get("Accept"))
```

Comment:
is it an intentional one?

Reply:
hehehehe no.

Reply:
removed in https://github.com/opendatahub-io/odh-model-controller/pull/362/commits/adc835d4f208425da4e0779dd0b0a152beb57da5.